docs/mesos/DockerizedDeployment.md

```diff
@@ -0,0 +1,209 @@
+# Deploying a Dockerized Mesos Cluster with Calico.
+
+In these instructions, we will run all cluster services as [Docker][docker] containers.  This speeds deployment and will prevent pesky issues like incompatible dependencies.  The services we will need are
+
+ * Zookeeper
+ * etcd
+ * Mesos Master
+ * Mesos Agent
+ * Calico
+ * Marathon (Mesos framework)
+
+In these brief instructions, I'll concentrate on getting Mesos and Calico up and running as quickly as possible.  This means leaving out the details of how to configure higly-available services.  Instead, we'll install Zookeeper, etcd, and the Mesos Master on the same "master" node.
+
+
+# Preparation
+
+These instructions are designed to run on CentOS/Red Hat Enterprise Linux 7, but other than the initial commands to configure Docker, should work on any Linux distribution that supports Docker 1.7+ and `systemd`.
+
+If your distribution does not support `systemd`, you will need to create initialization files for each of the services.  These should be straightforward based on the included `.service` files, but talk to us on the [Calico Users' Slack](https://calicousers-slackin.herokuapp.com/) if you want some assistance.  If you write init files for a new system, share the love!  PRs are welcome :) 
+
+## Install Docker
```

Comment:
I need to think about this more, so don't jump on this yet, but -

It doesn't make much sense that we're instructing users on how to install docker on their own systems. Realistically we should just link them to the docker install information for centos. This step made sense within Spike's guide as he was covering every single step between bare metal and a running cluster. But in these guides, I'm not sure it makes as much sense. 

The only reason I'm cautious to remove it is that the official docker docs don't mention anything about `docker-selinux`, as its a centos maintained rpm. And I can't find any documentation from centos covering how to install it...